app/views/we_eat/index.html.erb

```diff
@@ -1,15 +1,29 @@
 <p id="notice"><%= notice %></p>
 
-<h1>Your Pragmatic Catalog</h1>
+<h1>We Eat</h1>
 
-<% cache @restaurants do %>
-  <% @restaurants.each do |restaurant| %>
-    <% cache restaurant do %>
-      <div class="entry">
-        <h3><%= restaurant.name %></h3>
-        <label for="rating_paragraph">Rating</label>
-        <p id="rating_paragraph"><%= restaurant.rating %></p>
-      </div>
+<table>
```

Comment:
We can live with that in the pet project, but just FYI, html tables are not *that* common in day to day use, there are specific cases where you'd use a table
http://webdesignfromscratch.com/html-css/html-tables/